engine/src/main/java/org/terasology/rendering/world/RenderableWorldImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014 MovingBlocks
+ * Copyright 2016 MovingBlocks
```

Comment:
Watch out, I'm talking about the "RenderableWorld" instance, not the WorldRenderer. =)

But I'll have a look if by any chance also the WorldRenderer could be turned into a node...